hackclock/app/globals.css

```
@@ -0,0 +1,27 @@
+  --background: #0D1117;
+  --foreground: #E6EDF3;
+  
+  /* Custom Panel & Accent Colors */
+  --panel: #161B22;
+  --border: #30363D;
+  --text-muted: #8B949E;
+  --neon-blue: #4493F8;
+  --neon-green: #3FB950;
+}
+
+@theme inline {
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)